services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -116,6 +125,34 @@ export const DetailSection = ({
                 </Review>
               )
           )}
+          {detail.reviewTeam && (
+            <Review key="srt" heading="Review Team (SRT)">
+              {detail.reviewTeam.length === 0
+                ? "-- --"
+                : showReviewTeam
+                ? detail.reviewTeam.map((reviewer) => (
+                    <>
+                      {reviewer}
+                      <br />
+                    </>
+                  ))
+                : detail.reviewTeam.slice(0, 3).map((reviewer) => (
+                    <>
+                      {reviewer}
+                      <br />
+                    </>
```

Comment:
Im not a fan of the repeated block to show the reviewer........but i guess the only way around that would be to always do a slice with the second parameter being a dynamic var that is toggled between list size and NUM_REVIEWERS_TO_SHOW when the show/hide is toggled along with the showReviewTeam boolean. As is I think the slice should use the declared var and not a hard coded 3.

Reply:
I am also not a fan of the repeated block - but I was having SO MUCH trouble getting the logic right when I was "being clever" that I decided getting it right was more important than pretty :)   Though I absolutely agree about the slice 3 should use the constant - my only defense is that I did the constant *after* doing the slice and simply forgot... :O

Reply:
Apparently I channeled George this morning... sorry for making it a bigger re-review... but figured an internal component would isolate out the toggle and also remove the duplication... of course, I made a bunch of silly mistakes about which length was which in the process... but I *think* it is right now!